pages/product/index.vue

```diff
@@ -1,313 +1,344 @@
   SfDivider,
-  SfRadio,
   SfList,
   SfSelect,
   SfCheckbox,
   SfInput,
   SfAccordion,
+  SfLoader,
 } from "@storefront-ui/vue"
 
 import { defineComponent, computed } from "@vue/composition-api"
 
 import LazyHydrate from "vue-lazy-hydration"
 
 import { useAsync } from "@nuxtjs/composition-api"
+import KiboFulfillmentOptions from "@/components/KiboFulfillmentOptions.vue"
+import KiboProductActions from "@/components/KiboProductActions.vue"
 import { useProductSSR } from "@/composables/useProductSSR"
 import { productGetters } from "@/composables/getters"
 
 export default defineComponent({
   name: "Product",
   components: {
     SfGallery,
-    SfHeading,
     SfPrice,
     SfRating,
     SfIcon,
     SfButton,
-    SfAddToCart,
     SfBreadcrumbs,
+    SfLoader,
     LazyHydrate,
     SfColor,
     SfDivider,
-    SfRadio,
     SfList,
     SfSelect,
     SfCheckbox,
     SfInput,
     SfAccordion,
+    KiboFulfillmentOptions,
+    KiboProductActions,
   },
 
   setup(_, context) {
     const { productCode } = context.root.$route.params
-    const { load, product, configure } = useProductSSR(productCode)
+    const { load, product, configure, loading, error } = useProductSSR(productCode)
 
     useAsync(async () => {
       await load(productCode)
     }, null)
 
+    const productName = computed(() => productGetters.getName(product.value))
     const description = computed(() => productGetters.getDescription(product.value))
+    const shortDescription = computed(() => productGetters.getShortDescription(product.value))
     const breadcrumbs = computed(() => productGetters.getBreadcrumbs(product.value))
     const productGallery = computed(() => productGetters.getSFProductGallery(product.value))
 
-    const priceRegular = computed(() => productGetters.getPrice(product.value))
-    const priceSpecial = computed(() => productGetters.getSalePrice(product.value))
     const rating = computed(() => productGetters.getRating(product.value))
     const totalReviews = computed(() => productGetters.getProductTotalReviews())
     const properties = computed(() => productGetters.getProperties(product.value))
     const options = computed(() => productGetters.getOptions(product.value))
     const productOptions = computed(() => productGetters.getSegregatedOptions(product.value))
 
-    const addToCart = () => {}
-
+    // Options section
     const selectOption = async ({ value }, { attributeFQN }) => {
       await configure({ value, attributeFQN }, product.value?.productCode, options.value)
     }
 
+    // Add to Cart
+    const quantityLeft = computed(() => 5)
+    const qtySelected = useState(`pdp-selected-qty`, () => 1)
+
+    const addToCart = () => {
+      console.log("Add to Cart qunatity: ", qtySelected.value)
+    }
+    const addToWishList = () => {
+      console.log("Add to Whislist qunatity: ", qtySelected.value)
+    }
+
+    // Fullfillment Options
+    const fulfillmentOptions = useState(`pdp-fulfillment-options`, () => [
```

Comment:
Get fulfillmentTypesSupported in GQL query
`query { product { fulfillmentTypesSupported } }`

Reply:
Resolved